adapter/slack.py

```diff
@@ -177,5 +183,38 @@ def bot_replied(self, event, parent_message):
         
         if chatbot is None:
             return {"text": self.MISSING_CHATBOT_ERROR}
-                
-        return asyncio.run(self.process_chatbot_request(chatbot, question, channel_id, thread_ts))
+        
+        history = self.get_chat_history(event)
+        return asyncio.run(self.process_chatbot_request(chatbot, question, channel_id, thread_ts, history))

```

Comment:
I'm curious, when you return `asyncio.run`, what will be returned? Will it return the result of running `process_chatbot_request`?

Reply:
Yes, when you use asyncio.run() to execute an asynchronous function like process_chatbot_request, it returns the final result of that function.